notebooks/erroranalysis-dashboard-multiclass.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
Please add it to the notebook tests as well

Reply:
done!